spec.html

```diff
@@ -36755,6 +36755,17 @@ <h1>Structured Data</h1>
   <emu-clause id="sec-arraybuffer-objects">
     <h1>ArrayBuffer Objects</h1>
 
+    <emu-clause id="sec-arraybuffer-notation">
+      <h1>Notation</h1>
+      <p>The descriptions below in this section, <emu-xref href="#sec-atomics-object"></emu-xref>, and <emu-xref href="#sec-memory-model"></emu-xref> use the read-modify-write modification function internal data structure.</p>
+      <p>A <em>read-modify-write modification function</em> is a mathematical function that is notationally represented as an abstract closure that takes two Lists of bytes as arguments returns a List of bytes. These abstract closures satisfy all of the following properties:</p>
```

Comment:
```suggestion
      <p>A <em>read-modify-write modification function</em> is a mathematical function that is notationally represented as an abstract closure that takes two Lists of bytes as arguments and returns a List of bytes. These abstract closures satisfy all of the following properties:</p>
```
also, what is a "byte" in terms of spec or JS values? It doesn't seem defined in the current spec, but that seems like a big omission.

Reply:
It is a byte. I think that's a reasonable omission though, given prevalence of the shared understanding of "byte" in computing.

Reply:
"byte" or at least "byte-sized" is defined at the top of the section on [Data Blocks](https://deploy-preview-1907--ecma262-snapshots.netlify.com/#sec-data-blocks).

(FWIW there are a number of platforms with byte sizes other than 8 bits which are still in use. This comes up a lot on e.g. [the LLVM developers list](http://lists.llvm.org/pipermail/llvm-dev/2019-May/132080.html), where being imprecise about whether or not that is a shared assumption has led to a fair bit of pain. For a specification, it's worth mentioning. But I don't think we need to do more than we already have.)

Reply:
Added a sentence to the Data Blocks section.

Reply:
A byte can be a string of 0s and 1s, or an integer (mathematical, Number, BigInt), a char, or something else perhaps. I think it's important to be specific.